source/includes/_endpoint-progress.md

```diff
@@ -25,3 +20,8 @@ GET /progress/{id}/ HTTP/1.1
 }
 ```
 
+`GET` on a Progress view returns a Shoji View containing information about the status of the indicated process. The "progress" attribute contains a value between 0.0 and 100.0, stringified. If the `id` from the request URL does not exist, `GET` will nevertheless return 200 status and indicate `"progress": "100"`.
```

Comment:
Why do we return the progress value as a string? That seems silly and inconsistent with how we return numbers everywhere else in the API.